docs/eventing/accessing-traces.md

```diff
@@ -93,8 +93,7 @@ traces are provided in the Knative Serving observability section:
 ### Example
 
 The following demonstrates how to trace requests in Knative Eventing with Zipkin, using the
-[`TestBrokerTracing`](https://github.com/knative/eventing/blob/main/test/conformance/broker_tracing_test.go)
-End-to-End test.
+[`TestBrokerTracing`](https://github.com/knative/eventing/blob/master/test/conformance/broker_tracing_test.go) End-to-End test.
```

Comment:
It's main the default branch now, so let's leave this link as it is.

Reply:
ok I will close this PR.